src/components/Form.spec.js

```diff
@@ -9,7 +9,7 @@ describe('Form', () => {
     expect(screen.getByRole('button')).toBeInTheDocument()
   })
 
-  it('applies the movie title to the submit callback', () => {
+  it('applies the movie title and food category to the submit callback', () => {
```

Comment:
If you write it like this, I would expect the food category to be in you test. Maybe add add a second `userEvent` for this or leave it out of the test description for now.

Reply:
Forgot to delete it. I tried testing again today but it's not working properly. :(